srcpkgs/libqb/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'libqb'
 pkgname=libqb
-version=1.0.5
+version=2.0.0
 revision=1
 build_style=gnu-configure
+hostmakedepends="pkg-config libxml++-devel"
 short_desc="High performance logging, tracing, ipc, poll"
 maintainer="Kevin Klopfenstein <kk@sudo-i.net>"
```

Comment:
This is your name, but you're also listed as first time contributor?

Reply:
Nothing wrong if address is real, it just means address is not attached to gh account.